test_image.sh

```diff
@@ -69,6 +69,8 @@ function main {
 
 	test_health_status
 
+	test_docker_image_with_playwright
+
```

Comment:
@lucasmiranda0, can you please remove the blank line?

In addition, can you please try to sort this method alphabetically considering the method below named as `test_docker_image_*`?